week1/homework/src/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "src",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC"
+}
```

Comment:
This file is unnecessary. There's already a `package.json` in the `homework` directory.